common/python/ax/cluster_management/argo_cluster_manager.py

```diff
@@ -193,6 +199,15 @@ def install_platform_only(self, args):
         PlatformOnlyInstaller(platform_install_config).run()
         return
 
+    def uninstall_platform_only(self, args):
+        logger.info("Uninstalling Argo platform ...")
+        config.load_kube_config(args.kubeconfig)
+        api = client.CoreV1Api()
+        for namespace in ["axuser", "axsys"]:
+            api.delete_namespace(namespace, client.V1DeleteOptions())
```

Comment:
please add "axs3" to this list. what happens if a namespace doesn't exist?

Reply:
Added! And good point about namespace not existing. Added code to ignore if the namespace doesn't exist.